public/static/docs/glossary.js

```diff
@@ -13,6 +13,10 @@ code, ML models, etc. A workspace becomes a **DVC project** when
 [\`dvc init\`](/doc/command-reference/init) is run, and
 [DVC-files](/doc/user-guide/dvc-file-format) or stage files are created in it.
 
+Includes the
```

Comment:
it's kinda implied in the previous paragraph ... feels like we go too explicit here?

Reply:
It's only implied very broadly. I want to be extra clear of the relationship between Git concept "working tree" and DVC concept "workspace".

I'm planning to make some related updates on the core repo strings soon BTW – use "workspace" more.

Reply:
Sure. But this is too complicated. Term explanations should not be in 2-3 paragraphs, no one would read them.

Reply:
I don't know if I agree. If you'r reading the docs you probably have the interest to read a full term description. Where else could we clarify this relationship? Maybe once we have a basic concepts guide I can extract it there.

Reply:
p.s. see the Git Glossary: https://git-scm.com/docs/gitglossary. It has several long term descriptions.

Reply:
there is no one fits all solution for this, and I agree that in certain cases we should probably try to explain more. Two points though: we should them as tooltips - in general it's strange to see a very long text there, it's better to write a page and put a link in that case. In this particular case the concept is simple from the user perspective and we should be able to produce a single-paragraph description.

Reply:
OK, I rewrote the definition, please check. Thanks